### PR TITLE
Allow walkthroughs on modal view controllers

### PR DESCRIPTION
**Pull request by ChristopheFernandes**
@ruipfcosta I propose this update to allow walkthroughs on modal presented from root view controller

### CODE REVIEW

File: Sources/UIViewController+Walkthrough.swift [hunk 1]
```diff
@@ -11,7 +11,13 @@ import UIKit
 extension UIViewController: WalkthroughViewDelegate {
 
     public var rootController: UIViewController? {
-        return UIApplication.sharedApplication().delegate?.window??.rootViewController
+        let rootController = UIApplication.sharedApplication().delegate?.window??.rootViewController
+
+        if let presentedViewController = rootController?.presentedViewController {
```
Comment (ruipfcosta) [inline Sources/UIViewController+Walkthrough.swift]: Can we just change this to `return rootController?.presentedViewController ?? rootController`

**Review by ruipfcosta — CHANGES_REQUESTED** [review]

**Comment by ruipfcosta** [comment]
Thanks @ChristopheFernandes!

**Comment by ChristopheFernandes** [comment]
@ruipfcosta done ! ;)

File: Sources/Info.plist [hunk 2]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.0.3</string>
+	<string>0.0.4</string>
```
Comment (ruipfcosta) [inline Sources/Info.plist]: The version bump should be kept separate from this feature. Can you revert this change please?

File: SwiftyWalkthrough.podspec [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "SwiftyWalkthrough"
-  s.version      = "0.0.3"
```
Comment (ruipfcosta) [inline SwiftyWalkthrough.podspec:3]: Same comment as before regarding version bump.

**Review by ruipfcosta — CHANGES_REQUESTED** [review]

**Comment by ChristopheFernandes** [comment]
@ruipfcosta it's done.

**Review by ruipfcosta — APPROVED** [review]

**Comment by ruipfcosta** [comment]
It seems this wasn't that easy - have a look at https://github.com/ruipfcosta/SwiftyWalkthrough/commit/52b1072bb58950e8c72c20df662f1915a5c8d435 and check if suits your needs.

**Comment by ChristopheFernandes** [comment]
It's seems to be ok ! Sorry I can't test I didn't migrate my app to swift 3.0.

**Pull request merged**